hack/conformance.sh

```diff
@@ -12,11 +14,25 @@ fi
 export KUBECONFIG=$(cluster-up/kubeconfig.sh)
 
 teardown() {
+    rv=$?
```

Comment:
Please consider renaming thin value to `returnValue` or `errorCode`,
even though it is common in bash it is not trivial for new comers.

Reply:
i think we should stick with the common, new comers will learn it